### DISCUSSION TITLE
Error: Can not resolve config constant: 'ios'

### DISCUSSION THREAD

**Discussion by admit1243** [airsdk/Adobe-Runtime-Support#639] [Q&A]
## Environment:
AIR SDK: 33.0.2.246, 33.0.2.281, 33.0.2.288
OS: Android

## Problem Description
We are building our app through build script and it worked fine till we updated to AIR 33.

Getting "Error: Can not resolve config constant: 'ios'" while building android app with mxmlc command.

```
"$MXMLC \
  -dump-config=report_config.xml \
  -link-report=report_link.xml \
  -size-report=report_size.xml \
  -load-config=src/CompilerConfig.xml \
  -load-config=${FLEX_SDK}/frameworks/flex-config.xml \
  -source-path=src \
  -library-path=src/swc/common \
  -library-path=src/swc/android \
  -library-path=\"${FLEX_SDK}/frameworks/libs\" \
  -library-path=\"${FLEX_SDK}/frameworks/libs/air\" "
```

**Comment by ajwfrost** [comment]
Hi - this is strange as we've not been touching any of the files relating to building applications, these should be unchanged from the Adobe AIR SDK v32 releases.
The obvious thing to check is that the CompilerConfig.xml does define the 'ios' definition..
<define name="CONFIG::ios" value="false"/>
as you'd need to be adding this yourself somewhere..? Or using it on the command line with
-define+=CONFIG::ios,false
But afaik this definition isn't built in to the SDK anywhere; if it worked for you before please confirm the exact version number of the SDK that you have (look for air-sdk-description.xml and flex-sdk-description.xml files in the root of your SDK folder).
thanks
Are you able to reproduce it with a very simple test case stripping out all of the extra libraries etc, 
thanks
